sparkler-api/src/main/java/edu/usc/irds/sparkler/model/Resource.java

```diff
@@ -23,15 +23,15 @@
     //@Field private Integer numTries = 0;
     //@Field private Integer numFetches = 0;
     @Field("discover_depth") private Integer discoverDepth = 0;
-    @Field private Double score = 0.0;
+    @Field("page_score") private Double score = 0.0;
+    @Field("generate_score") private Double generateScore = 0.0;
```

Comment:
Good start, but this is still an incomplete solution.

## Why incomplete?
1. Scorer plugin can define any key (see, you have `getKey()`)  -- thats good, lets keep it
2. Solr schema can take many scoring fields, there is dynamic field with `_score` field. -- this is awesome, lets keep it.
3. Resource have two fields : `score` and `generate_score`. It can only take two scoring values -- this is incomplete


## Whats the problem? 
Lets say we add a new scorer plugin which produces a key `new_score`, 
1) Resource object cannot take it and
2) Resource object doesnt know that the score is called by field name `new_score`. 

## Solution:
Score field should be actually `Map<String, Double> score`.
Look at the discussion here for how to do it https://issues.apache.org/jira/browse/SOLR-1129
Also change the getters and setters accordingly. 
You can also take away the `score` and `generateScore` variables and just use the Map<String, Double> to store them -- so to avoid future confusions.